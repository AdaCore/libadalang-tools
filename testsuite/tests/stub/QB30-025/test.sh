rm -f *.ads *.adb
gnatchop -q -w source.ada
gnatstub --quiet --subunits p.ads
gnatmake -q -f -g -O0 -gnata p-main.adb
p-main
