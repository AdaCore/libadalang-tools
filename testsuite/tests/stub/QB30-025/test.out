p-q.adb:5:33: warning: Q unimplemented [enabled by default]
p-r.adb:5:33: warning: R unimplemented [enabled by default]
p-s.adb:5:33: warning: S unimplemented [enabled by default]

raised PROGRAM_ERROR : Unimplemented procedure Q
>>>program returned status code 1
